air/src/stack/op_flags/tests.rs

```diff
@@ -121,3 +121,410 @@ fn degree_4_op_flags() {
         }
     }
 }
+
+/// Composite flag unit tests.
+#[test]
+fn composite_flags() {
+    // ------ no change 0 ---------------------------------------------------------------------
+
+    let op_no_change_0 = [Operation::MpVerify, Operation::Span, Operation::Halt];
+    for op in op_no_change_0 {
+        // frame initialised with an op operation.
+        let frame = generate_evaluation_frame(op.op_code().into());
+
+        // All the operation flags are generated for the given frame.
+        let op_flags = OpFlags::new(&frame);
+
+        for i in op_flags.no_shift_flags.iter() {
+            assert_eq!(*i, ONE);
+        }
```

Comment:
nit: instead of dereferencing `i` as `*i`, you could re-write this as:
```Rust
for &i in op_flags.no_shift_flags.iter() {
    assert_eq!(i, ONE);
}
```